### ISSUE TITLE
Agregated mode is not so intuitive

### ISSUE THREAD

**Issue by phiphou** [electricitymaps/electricitymaps-contrib#5774]
Hi.

As you mays know, lot of people use your website or application to discuss about electricity politics and systems.

A lot of those who share screenshots in aggregated mode seems to misunderstand it. They click on the 12 months button, for instance, see a value in the colored square, and think this value is the mean of the past 12 months. 

The information on witch month is displayed is present (just above the colored square), but they seem to miss it.

It may be useful to think about another way to present this mode, don't you think ?

I've no idea on how to do it better, but it appears many people are concerned.

**Comment by phiphou** [comment]
We need an amount. The question is : may we add a separate global mean or not.

As I understand it, when people click on 12 months, they expect to see the average value for the past 12 months. 

But I also think it's a good feature to see all the monthly means, on witch we can navigate. 

So I think we need to present both, the actual behavior and a global mean.

**Comment by VIKTORVAV99** [comment]
I was thinking we just remove 24, 30, 12 and 6 from hours, days, months and years respectively and rename them to hourly, daily, monthly and yearly.

That way we don't specify a time period and the amounts shown will not be associated with one?

**Comment by phiphou** [comment]
It could be a solution.

But would it be possible to also have the period total average ?

The misunderstanding I talked about shows many users are looking for this information.

**Comment by VIKTORVAV99** [comment]
> It could be a solution.
> 
> But would it be possible to also have the period total average ?
> 
> The misunderstanding I talked about shows many users are looking for this information.

I don't think we want to show the average of all the days on the daily view (that is what the monthly is for and all monthly averages are on yearly etc). But perhaps we can re-work how this is presented with the aggregated label as well.

I think this is something we need to discuss more in depth but it will probably be after we have rolled out energy values (instead of power) for daily, monthly and yearly which is being worked on right now.

**Comment by tonypls** [comment]
Thanks for highlighting this @phiphou! 

I think it's a good point and we're not really following what is completely intuitive and standard across the web.

For example I think the challenge is that the label "30 days" correctly captures what is available in the slider but this doesn't reflect the information of the map and associated charts. 

Switching to the labels to "Hourly, Daily, Monthly and Yearly" could help and perhaps there is a feature we could develop to capture an average for the period, totally open for creative suggestions for this 🤔

**Comment by phiphou** [comment]
Do you think "Hourly, Daily, Monthly and Yearly" labels will fit the available space in all languages ?

**Comment by phiphou** [comment]
It works for English, German and French. Hope it will be the case for the rest...

![FireShot Capture 1873 - Electricity Maps - Émissions CO₂ de la consommation électrique en te_ - app electricitymaps com](https://github.com/electricitymaps/electricitymaps-contrib/assets/364418/a008cb78-4a4e-4d62-b470-04ab2dfdddef)

![FireShot Capture 1874 - Electricity Maps - Émissions CO₂ de la consommation électrique en te_ - app electricitymaps com](https://github.com/electricitymaps/electricitymaps-contrib/assets/364418/0768d084-d0bd-4a1d-b59b-7e60e05cbe8a)

![FireShot Capture 1875 - Electricity Maps - Émissions CO₂ de la consommation électrique en te_ - app electricitymaps com](https://github.com/electricitymaps/electricitymaps-contrib/assets/364418/3f3cdfe9-2d07-4424-b822-df19199e9348)

**Comment by phiphou** [comment]
What about the periods means ? 

I think it would be useful to have a mean for the past 30 days and the past 24 hours. 

May I open a specific issue for it ?

**Comment by VIKTORVAV99** [comment]
We are closing down some longstanding issues that have't had any activity in a while. If you believe this is still an issue let us know and we can reopen it.
